docs/learn.rst

```diff
@@ -260,6 +261,15 @@ To read digital books in Kolibri you have several available controls inside the
 
 .. _lessons:
 
+Slideshow Viewer options
+""""""""""""""""""""""""
+
+To navigate the slides of a slideshow, you may use the on-screen navigation controls or your keyboard's arrow keys.
+
+* Use the |fullscreen| button in the upper right corner to open the ebook in fullscreen view.
+* With the :guilabel:`Left Arrow` or :guilabel:`Right Arrow` keys or the buttons on the side of the slideshow.
```

Comment:
We use `:guilabel` to reference UI elements (buttons mostly) in Kolibri interface, and less often keyboard keys. Since these keys are not "spelled" out in words on the keyboard, and I don't think we can put the icon inside that directive, I'm hesitant if they should be used as-is. 

On another hand, we do have substitutions for `|next-page|` and `|previous-page|` used for ebook content, so we could reuse those or make a version styled as in the slideshow.

None of the above is a blocker, so feel free to pass, and I'll correct/add next week with the 0.12.6 update.